lib/superstore/types/base.rb

```diff
@@ -0,0 +1,9 @@
+module Superstore
+  module Types
+    class Base < ActiveModel::Type::Value
+      def type
+        self.class.name.demodulize.sub(/Type$/, '').downcase
```

Comment:
Shouldn't we get `geo_point` from `GeoPointType`?  
```suggestion
        self.class.name.demodulize.sub(/Type$/, '').underscore
```

Reply:
Yeah, makes sense. I'll add it to master.